source/extensions/filters/network/dubbo_proxy/conn_manager.cc

```diff
@@ -32,21 +31,6 @@ Network::FilterStatus ConnectionManager::onData(Buffer::Instance& data, bool end
 
   if (end_stream) {
     ENVOY_CONN_LOG(trace, "downstream half-closed", read_callbacks_->connection());
-
-    // Downstream has closed. Unless we're waiting for an upstream connection to complete a oneway
-    // request, close. The special case for oneway requests allows them to complete before the
```

Comment:
Why remove

Downstream half closed, does it need to wait for the response of the upstream?

Reply:
sorry, the code was deleted by mistake while merging, I have restored it.